### ISSUE TITLE
Re_index problem 

### ISSUE THREAD

**Issue by wwx13** [Quareia/LTA#4]
Hi!
I am in trouble now. Hope you can help me. Thank u!

```
trainer.py

class StepTwoTrainer(BaseTrainer):
  def _train_epoch(self, epoch):

      re_index = torch.sort(torch.cat([seen_class_idxs, unseen_class_idxs], 0).to(self.device))[1]           
      unseen_querys_y = re_index[unseen_querys_y]
      seen_querys_y = re_index[seen_querys_y]

```

I think you are reindex original label id to 0-(num_select_seen_class+num_fakeunseen_class). But these operations cant realize as we wish.  How can we index a label id in re_index variable which is the sorted indices of selected labels?

**Comment by Quareia** [comment]
You can find the source code of `torch.sort` and realize your own `torch.sort`, maybe it's the most convenient way to realize it.

**Comment by wwx13** [comment]
> You can find the source code of `torch.sort` and realize your own `torch.sort`, maybe it's the most convenient way to realize it.

Ok, thanks.  I know how to do now.
